README.md

```diff
@@ -1,3 +1,71 @@
-# Encryptonize Go Client Library v3.2.0
+# Go Client Library for Encryptonize&reg;
 
-A simple Encryptonize go client library.
+Go client libraries for
+* [Encryptonize&reg; Core](https://github.com/cyber-crypt-com/encryptonize-core)
+* [Encryptonize&reg; Objects](https://github.com/cyber-crypt-com/encryptonize-objects)
+* [Encryptonize&reg; Key Server](TODO)
+
+## Usage
+The easiest way to use the Encryptonize&reg; Core client is through the `NewCoreClientWR`
+constructor. The resulting client will automatically refresh the user's access token when it
+expires. For details on how to create a user, see the Encryptonize&reg; Core [user
+manual](https://github.com/cyber-crypt-com/encryptonize-core/blob/master/documentation/user_manual.md).
+
+```go
+// Create a new Encryptonize Core client, providing the hostname, a root CA certificate, and user
+// credentials.
+client, err := NewCoreClientWR(context.Background(), "localhost:9000", "./ca.crt", "user id", "password")
```

Comment:
Some explanation about how to obtain the user credentials (and/or a link to the explanation in the core documentation) would be nice to have.

Reply:
There's a link right above this code block, do you want something other than that?

Reply:
That's a link to the whole manual. If I want to use this client, I would like to have all the information related to configuring it in one place, rather than having to search through the manual for the specific info. Plus bootstrapping the initial user can be confusing and I think we should say that here.

Maybe saying in the previous paragraph sth like:
When configuring the client for the first time, you need to bootstrap an initial user with credentials either through the executable as described [here](https://github.com/cyber-crypt-com/encryptonize-core/blob/master/documentation/user_manual.md#bootstrapping-users), or through the API as described [here](https://github.com/cyber-crypt-com/encryptonize-core/blob/master/documentation/user_manual.md#creating-users-through-the-api).

Reply:
Fixed